third_party/blink/renderer/bindings/core/v8/v8_script_runner.cc

```diff
@@ -824,6 +824,18 @@ ScriptEvaluationResult V8ScriptRunner::EvaluateModule(
     // only after module error handling to ensure proper timing with and
     // without top-level await.
 
+    absl::optional<recordreplay::AutoDependencyExecution> execute;
+    if (recordreplay::DependencyGraphEnabled()) {
+      base::Value::Dict info;
+      info.Set("kind", "evaluateModule");
+      if (module_script)
+        info.Set("url", module_script->SourceUrl().GetString().Utf8());
+      std::string json;
+      base::JSONWriter::Write(info, &json);
+      int node_id = recordreplay::NewDependencyGraphNode(json.c_str());
+      execute.emplace(node_id);
+    }
```

Comment:
This is purely a copy-paste (with small adjustments) of the code here: https://github.com/replayio/chromium/blob/656a07519214d4f49536cf0a08a3a66d175d7e6f/third_party/blink/renderer/core/script/pending_script.cc#L192-L207